src/wp-content/themes/twentytwenty/index.php

```diff
@@ -31,9 +31,9 @@
 		global $wp_query;
 
 		$archive_title = sprintf(
-			'%1$s %2$s',
+			'%1$s &ldquo;%2$s&rdquo;',
```

Comment:
The quotation marks probably should have been translatable.

```suggestion
			/* translators: Main heading for the Search results template. 1: "Search:" text, 2: search query. */
			__( '%1$s &ldquo;%2$s&rdquo;', 'twentytwenty' ),
```